scripts/mk.js

```diff
@@ -4746,6 +4746,14 @@ function startGame() {
 				if (!pause || !fInfos.replay) {
 					function handleInputPressed(gameAction) {
 						switch (gameAction) {
+							case "item":
+							case "item_back":
+							case "item_fwd":
+								if (!oPlayers[0].tourne && !oPlayers[0].cannon && !pause && !oPlayers[0].justPressedItem) {
+									oPlayers[0].justPressedItem = true;
+									arme(0, ("item_back" === gameAction), ("item_fwd" === gameAction));
+								}
+								break;
```

Comment:
Is it something that has been requested by the community?
I like the way it works today, it makes it clear that you have to press the key twice to use an item (once to put it behind you want once to launch it), which can be confusing when you come from the official games
(+ it makes the code simpler 😄)